org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/FormatterHandlerTest.java

```diff
@@ -61,6 +61,7 @@ public void setUp() {
 		javaProject = JavaCore.create(project);
 		originalTabChar = javaProject.getOption(DefaultCodeFormatterConstants.FORMATTER_TAB_CHAR, true);
 		javaProject.setOption(DefaultCodeFormatterConstants.FORMATTER_TAB_CHAR, JavaCore.TAB);
+		javaProject.setOption(DefaultCodeFormatterConstants.FORMATTER_INDENT_SWITCHSTATEMENTS_COMPARE_TO_SWITCH, DefaultCodeFormatterConstants.TRUE);
```

Comment:
why do we need to set it manually when we want to check the effect of jdt.ls default formatting options? If anything we should ensure PreferenceManager.initialize() was called

Reply:
Fixed.